testnet/stacks-node/src/event_dispatcher.rs

```diff
@@ -214,12 +215,14 @@ impl EventObserver {
             }
         };
 
-        let (txid, raw_tx) = match tx {
-            TransactionOrigin::Burn(txid) => (txid.to_string(), "00".to_string()),
+        let (txid, raw_tx, burnchain_op_json) = match tx {
+            TransactionOrigin::Burn(op) => {
+                (op.txid().to_string(), "00".to_string(), json!(op.clone()))
```

Comment:
The default JSON encoding of a `BlockstackOperationType` is pretty unfriendly (I modified it for a specific op type in this PR: #3511 -- so you can look there for what the schema would be without any changes).

@zone117x -- would it be sufficient to just supply the whole bitcoin transaction as a hex string here, or would you prefer something like the encoding used in that PR? It looks like this:

```json
{
  "PegIn": [
    {
      "amount": 1337,
      "block_height": 217,
      "burn_header_hash": "66b2f18530ccf0e57e82dd054cca9fa41bf5f645f7757171ad9c1546b86387e1",
      "peg_wallet_address": "tb1pqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqkgkkf5",
      "recipient": "S0000000000000000000002AA028H",
      "txid": "84808f747841fb26e74f171487e56eda1a51373581eb04a8c87a421a77bd46d2",
      "vtxindex": 1
    }
  ]
}
```

Reply:
The parsed / expanded data like the above sample looks good. It would also be helpful to include the full bitcoin tx hex string as well.

Reply:
If it's not too much trouble to adapt @kantai's work from #3511 to all the other burnchain ops, then we should just bite the bullet and go for it.

Reply:
done